File: app/src/main/java/com/chooloo/www/callmanager/ui/listitem/ListItemHolder.java
package com.chooloo.www.callmanager.ui.listitem;

import android.content.Context;

import androidx.annotation.NonNull;
import androidx.recyclerview.widget.RecyclerView;

public class ListItemHolder extends RecyclerView.ViewHolder {

    private final ListItem mListItem;

    public ListItemHolder(Context context) {
        this(new ListItem(context));
    }

    public ListItemHolder(@NonNull ListItem itemView) {
        super(itemView);
        mListItem = itemView;
    }

    public ListItem getListItem() {
        return mListItem;
    }
}

File: app/src/main/java/com/chooloo/www/callmanager/ui/widgets/ListItemHolder.kt
package com.chooloo.www.callmanager.ui.widgets

import android.content.Context
import androidx.recyclerview.widget.RecyclerView
import com.chooloo.www.callmanager.ui.widgets.ListItem

class ListItemHolder(
        val listItem: ListItem
) : RecyclerView.ViewHolder(listItem) {

    constructor(context: Context) : this(ListItem(context))
}